commands/load_balancers.go

```diff
@@ -445,6 +446,12 @@ func buildRequestFromArgs(c *CmdConfig, r *godo.LoadBalancerRequest) error {
 	}
 	r.SizeUnit = uint32(sizeUnit)
 
+	lbType, err := c.Doit.GetString(c.NS, doctl.ArgLoadBalancerType)
+	if err != nil {
+		return err
+	}
+	r.Type = lbType
```

Comment:
From the consts in godo, it looks like this value needs to be in all capital letters:

https://pkg.go.dev/github.com/digitalocean/godo#LoadBalancerTypeGlobal

Unless the backend is already doing something similar, it might be nice for users if we do something like:

```suggestion
	r.Type = strings.ToUpper(lbType)
```

Reply:
Backend is already doing something similar: ie it accepts case-insensitive versions of `GLOBAL` and `REGIONAL`, and godo consts have been created to make it more intuitive to the users.

Reply:
Addressed in 8c8a89ccaf1ac055d751b98ca380f68330f12f99.